.storybook/static/data/afrique/bbc_afrique_radio/schedule.json

```diff
@@ -0,0 +1,3124 @@
+        }
+      },
+      "version": { "pid": "w1msh7q0dtj96n4" }
+    },
+    {
+      "urn": "urn:bbc:ares:ws_media:schedule:bbc_afrique_radio",
+      "serviceId": "bbc_afrique_radio",
+      "partnerId": "s0000001",
+      "updateTime": 1583742373000,
+      "publishedTimeStart": 1584691200000,
+      "publishedTimeEnd": 1584691320000,
+      "publishedTimeDuration": "PT2M",
+      "transmissionTimeStart": 1584691200000,
+      "transmissionTimeEnd": 1584691320000,
+      "isBlanked": false,
+      "isRepeat": false,
+      "isAudioDescribed": false,
+      "isCritical": false,
+      "isSimulcast": true,
+      "broadcast": {
+        "bid": "w1mshbfj9d0tydh.imi:ws.bbc.co.uk/FRENA1584691200",
+        "pid": "p0862rpj"
+      },
+      "brand": {
+        "pid": "p030s6dq",
+        "title": "Bulletin D'informations",
+        "image": "ichef.bbci.co.uk/images/ic/$recipe/p035djqv.jpg",
+        "synopses": {
+          "short": "Le tour du monde de l'actualité en 2 minutes",
+          "medium": "Le tour du monde de l'actualité en 2 minutes"
+        }
+      },
+      "episode": {
+        "pid": "w172x5qw84sng9y",
+        "presentationTitle": "20/03/2020 GMT",
+        "image": "ichef.bbci.co.uk/images/ic/$recipe/p035djqv.jpg",
+        "synopses": {
+          "short": "Le tour du monde de l'actualité en 2 minutes",
+          "medium": "Le tour du monde de l'actualité en 2 minutes"
+        }
+      },
+      "version": { "pid": "w1mshbfj9d0tydh" }
+    },
+    {
+      "urn": "urn:bbc:ares:ws_media:schedule:bbc_afrique_radio",
+      "serviceId": "bbc_afrique_radio",
+      "partnerId": "s0000001",
+      "updateTime": 1583745945000,
+      "publishedTimeStart": 1584694800000,
+      "publishedTimeEnd": 1584694920000,
+      "publishedTimeDuration": "PT2M",
+      "transmissionTimeStart": 1584694800000,
+      "transmissionTimeEnd": 1584694920000,
+      "isBlanked": false,
+      "isRepeat": false,
+      "isAudioDescribed": false,
+      "isCritical": false,
+      "isSimulcast": true,
+      "broadcast": {
+        "bid": "w1mshbfj9d0v24m.imi:ws.bbc.co.uk/FRENA1584694800",
+        "pid": "p0862wkg"
+      },
+      "brand": {
+        "pid": "p030s6dq",
+        "title": "Bulletin D'informations",
+        "image": "ichef.bbci.co.uk/images/ic/$recipe/p035djqv.jpg",
+        "synopses": {
+          "short": "Le tour du monde de l'actualité en 2 minutes",
+          "medium": "Le tour du monde de l'actualité en 2 minutes"
+        }
+      },
+      "episode": {
+        "pid": "w172x5qw84snl22",
+        "presentationTitle": "20/03/2020 GMT",
+        "image": "ichef.bbci.co.uk/images/ic/$recipe/p035djqv.jpg",
+        "synopses": {
+          "short": "Le tour du monde de l'actualité en 2 minutes",
+          "medium": "Le tour du monde de l'actualité en 2 minutes"
+        }
+      },
+      "version": { "pid": "w1mshbfj9d0v24m" }
+    },
+    {
+      "urn": "urn:bbc:ares:ws_media:schedule:bbc_afrique_radio",
+      "service
```

Comment:
Other services have been updated to set one of the schedules to have a PresentationTitle `01/01/2030 09:00 GMT` and timestamps of the same date to keep the data up to date. Wondering if the same should be done here and in the local data as well

Reply:
Good point Ruth!. Will update accordingly